docs/snippets/common/main-config-swc-jsx-transform.ts.mdx

```diff
@@ -0,0 +1,24 @@
+```ts
+// Replace your-framework with the webpack-based framework you are using (e.g., react-webpack5)
+import type { StorybookConfig } from '@storybook/your-framework';
+
+const config: StorybookConfig = {
+  framework: {
+    name: '@storybook/your-framework',
+    options: {
+      builder: {
+        useSWC: true, // This flag is automatically set by Storybook for all new Webpack5 projects (except Angular) in Storybook 7.6
+      },
+    },
+  },
+  swc: (config, options) => ({
+    jsc: {
+      transform: {
+        runtime: 'automatic',
+      },
+    },
```

Comment:
Sorry, I gave you the wrong code snippet. It has to be:

```suggestion
    jsc: {
      transform: {
        react: {
          runtime: 'automatic'
        }
      }
    }
```

Reply:
No problem whatsoever. Thanks for clarifying it. I've updated them with the latest commit.